components/lib/crash/src/main/java/mozilla/components/lib/crash/handler/CrashHandlerService.kt

```diff
@@ -49,10 +49,7 @@ class CrashHandlerService : Service() {
             val channel = CrashNotification.ensureChannelExists(this)
             val notification = NotificationCompat.Builder(this, channel)
                 .setContentTitle(
-                    getString(
-                        R.string.mozac_lib_crash_dialog_title,
-                        crashReporter.promptConfiguration.appName
-                    )
+                    getString(R.string.mozac_lib_gathering_crash_data_in_progress)
```

Comment:
For context, what was the intent behind changing the title?

Reply:
The original string was confusing users thinking that we were sending the crash report automatically.  This is not the case, we were just collecting the crash information to store locally for user to look at in "about:crashes" and in case we failed to send it with crash reporter.